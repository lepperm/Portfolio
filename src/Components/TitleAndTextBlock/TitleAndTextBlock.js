/**
 * @fileoverview TitleAndTextBlock component.
 */

import React from "react";
import PropTypes from "prop-types";

/**
 * TitleAndTextBlock component
 *
 * @return {Component} TitleAndTextBlock component
 */
export const TitleAndTextBlock = () => {
  return (
    <div>
      <header>
        <h2>Title</h2>
      </header>
      <p>The text explaining and supporting the section</p>
    </div>
  );
};

/**
 * Prop types for TitleAndTextBlock component
 */
TitleAndTextBlock.propTypes = {
  //
};
