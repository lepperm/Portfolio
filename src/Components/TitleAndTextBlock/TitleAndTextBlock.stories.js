/**
 * @fileoverview Story for TitleAndTextBlock component.
 */

import React from "react";
import { TitleAndTextBlock } from "./TitleAndTextBlock";

export default {
  title: "TitleAndTextBlock",
  component: TitleAndTextBlock,
};

export const Basic = () => <TitleAndTextBlock />;
