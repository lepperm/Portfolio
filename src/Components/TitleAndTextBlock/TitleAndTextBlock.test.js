/**
 * @fileoverview Tests for TitleAndTextBlock component.
 */

import React from "react";
import { shallow } from "enzyme";
import { TitleAndTextBlock } from "./TitleAndTextBlock";

describe("TitleAndTextBlock", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<TitleAndTextBlock />);
    expect(wrapper).toMatchSnapshot();
  });
});
