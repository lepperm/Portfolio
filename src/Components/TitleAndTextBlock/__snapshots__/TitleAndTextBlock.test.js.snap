// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleAndTextBlock matches snapshot 1`] = `ShallowWrapper {}`;
